anyTV/quasar-boilerplate
# Delta-1642 Platform > Freedom Components > FHistory

## Ticket references
- [Delta-1642 Platform > Freedom Components > FHistory](https://freedom.myjetbrains.com/youtrack/issue/Delta-1642)

## Summary of changes
- added FHistory demo page in quasar boilerplate


## Checklist


## Testing
  - Use this [branch](https://gitlab.com/anyTV/incubator/freedom-components/-/merge_requests/62) to use the fhistory component
  - Apply this diff `curl https://gist.githubusercontent.com/paulyves/33e1c8a70fb38a6bdcb9bbe612162cf4/raw | git apply`
  - remove package-lock.json and run `npm install`
  - Check the [fhistory-demo](https://localhost:8080/fhistory-demo) page and see all the statuses that the status_types props supported.
  - Click the item with long notes to collapse


![image](https://user-images.githubusercontent.com/17078489/155257545-41552645-e377-4356-b2b7-61abb63595ff.png)
